cpu/idle/utilsetup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
This script doesn't seem to be called anywhere, nor is it a part of `FILES` list in the makefile.

Reply:
@veruu, the script is removed and added its code to `runtest.sh`. If test system can install package `msr-tools` successfully via Makefile, we just use the system's setup, else we try to install msr-tools via its source code. Please refer to function `msr_tools_setup()` and `msr_tools_cleanup()` in file `runtest.sh`.

Here is a manual testlog, 
- http://pastebin.test.redhat.com/776485

Please help to review again, thanks!